src/test/java/it/com/semmle/jira/addon/IntegrationTest.java

```diff
@@ -72,17 +75,18 @@ public IntegrationTest() throws JSONException, ClientProtocolException, IOExcept
   }
 
   @Test
-  public void testCreationAndTransitionOfIssues() throws IOException, JSONException {
-    // Create three issues, subsequent tests assume these have known IDs
-    Issue issueA = postWebhookJson(CREATE_STRING, CREATE_SIGNATURE);
+  public void testCreationAndTransitionOfIssues() throws IOException {
+    Request createRequest = Util.JSON.readValue(CREATE_STRING, Request.class);
```

Comment:
This could be done in the `@Before` so the createRequest is available for all tests